app/Http/Middleware/UploadCheck.php

```diff
@@ -66,8 +66,8 @@ public function handle(Request $request, Closure $next)
      */
     public function album_check(Request $request, int $id)
     {
-        if ($request->has('albumID')) {
-            $albumID = $request['albumID'];
+        if ($request->has('albumID') || $request->has('parent_id')) {
+            $albumID = $request[$request->has('albumID') ? 'albumID' : 'parent_id'];
```

Comment:
While I see how this works and the beauty for this ternary, I think it is slightly too complex to read. I think this would be easier to read:
```php
$albumID = $request->has('albumID') ? $request[ 'albumID'] : $request['parent_id'];
```

Reply:
Sure, no problem. I'm not emotionally attached to that ternary :wink: